qemu.go

```diff
@@ -468,13 +476,20 @@ func (q *qemu) setCPUResources(podConfig PodConfig) ciaoQemu.SMP {
 	return smp
 }
 
-func (q *qemu) setMemoryResources(podConfig PodConfig) ciaoQemu.Memory {
+func (q *qemu) setMemoryResources(podConfig PodConfig) (ciaoQemu.Memory, error) {
+	hostMemBytes, err := getHostMemorySizeBytes("/proc/meminfo")
```

Comment:
`/proc/meminfo` is a very well known path for Linux. Please define this as a const and don't provide it through this call, instead use it directly from the function `getHostMemorySizeBytes()`.